Signal/src/ViewControllers/OWSConversationSettingsViewController.m

```diff
@@ -171,7 +171,7 @@ - (BOOL)hasExistingContact
     OWSAssert([self.thread isKindOfClass:[TSContactThread class]]);
     TSContactThread *contactThread = (TSContactThread *)self.thread;
     NSString *recipientId = contactThread.contactIdentifier;
-    return [self.contactsManager.lastKnownContactRecipientIds containsObject:recipientId];
+    return [self.contactsManager signalAccountForRecipientId:recipientId] != nil;
```

Comment:
We do this a few places.  I'd make a `hasSignalAccountForRecipientId` method personally to yield more readable code, but I know many people prefer to avoid the redundancy. TIOLI